_sass/custom/lessons.scss

```diff
@@ -3,3 +3,10 @@ mjx-container {
   overflow-y: hidden;
   max-width: 100%;
 }
+
+details{
+    padding-left:1%;
+    border-left: 2px solid #F3715B;
```

Comment:
Use a color that already exists in variables or make a new one with a descriptive name (for custom theme support)